charts/posit-chronicle/values.yaml

```diff
@@ -0,0 +1,85 @@
+image:
+  repository: "935931255537.dkr.ecr.us-east-2.amazonaws.com/chronicle"
+  tag: "latest"
+  imagePullPolicy: "Always"
+
+serviceaccount:
+  enabled: false
+  # -- Additional annotations to add to the chronicle-server serviceaccount
+  annotations: {
+    # EKS role arn example
+    # eks.amazonaws.com/role-arn: my-aws-iam-role-arn
+  }
+  # -- Additional labels to add to the chronicle-server serviceaccount
+  labels: {}
+
+service:
+  rest:
+    # -- The port to use for the REST service
+    port: 80
+    # -- The port to forward REST requests to on the pod. Also see pod.port
+    targetPort: 5252
+    # -- Additional annotations to add to the chronicle-server service
+    annotations: {}
+    # -- Additional labels to add to the chronicle-server service
+    labels: {}
+
+# -- The number of replica pods to maintain for this service
+replicas: 1
+
+pod:
+  command: "/chronicle"
+  args: ["start", "-c", "/opt/chronicle/config.gcfg"]
+  rest:
+    port: 5252
+  # -- Optional environment variables
+  env: []
+  # -- Additional annotations to add to the chronicle-server pods
+  annotations: {}
+  # -- Additional labels to add to the chronicle-server pods
+  labels: {}
+  # -- A map used verbatim as the pod's "affinity" definition
+  affinity: {}
+  # -- A map used verbatim as the pod's "nodeSelector" definition
+  nodeSelector: {}
+  # -- An array used verbatim as the pod's "tolerations" definition
+  tolerations: []
+
+# We do want at least 1Gi of persistent volume space
+# to perform compaction/compression operations
+storage:
+  persistentVolumeSize: 1Gi
+
+config:
+  http:
+    listen: ":5252"
+  https:
+    # If https.enabled=true, we will ignore any http
+    # values and enable https in our config instead
+    enabled: false
+    listen: ":443"
+    key: ""
+    certificate: ""
+  localStorage:
+    enabled: true
+    location: "./chronicle-data"
+    retentionPeriod: "30d"
+  s3Storage:
+    enabled: false
+    bucket: "posit-chronicle"
+    prefix: ""
+    profile: ""
+    region: "us-east-2"
+    compactionEnabled: false
+  metrics:
+    enabled: true
+  profiling:
+    enabled: false
+    listen: :3030
+  tracing:
+    enabled: false
+    address: ""
```

Comment:
Since there's no facility for getting this data out right now and these are hidden in the default config, I'd suggest removing these entirely so they don't get picked up by customers copy & pasting this `values.yaml` file (which happens regularly).

```suggestion
```

Reply:
I'm definitely for to removing the tracing portion, the port definitions i'm not sold on yet but will consolidate to another thread

Reply:
I'm removing all the port definitions as we discussed in meeting 👍

i think we should keep the profiling bit though, i've actually made use of pprof a few times in we'll want to continue using it for our longer term memory utilization QA